packages/metascraper-image/index.js

```diff
@@ -21,6 +21,8 @@ const getSrc = el => el.attr('src')
  */
 module.exports = () => ({
   image: [
+    wrap($jsonld('image.0')),
```

Comment:
@plaa 

Your original code was suggested image could be an object with `url` property inside.

I didn't find a JSON-LD example following that structure. However, I found it could be an array as well https://github.com/schemaorg/schemaorg/wiki/Examples:-Photos-and-Images

Reply:
Ok I'm seeing at schema.org it could be an object:https://schema.org/image

but It doesn't say anything about array 🤔